testApi/src/main/kotlin/testApi/testRunner/TestRunner.kt

```diff
@@ -197,6 +200,48 @@ abstract class AbstractCoreTest {
     }
 }
 
+abstract class AbstractKDocTest : AbstractCoreTest() {
```

Comment:
IMHO this should not be in the testAPI as those methods are meant for a specific type of KDoc tests. That's why this should be moved to a separate file in core/test/markdown

Reply:
Moved